setup.py

```diff
@@ -173,7 +173,9 @@ def BTreeExtension(family):
       url="https://github.com/zopefoundation/BTrees",
       license="ZPL 2.1",
       platforms=["any"],
-      packages=find_packages(),
+      # Make sure we don't get 'terryfy' included in wheels
+      # created on macOS CI
+      packages=find_packages(include=("BTrees",)),
```

Comment:
This is BTW a yet another reason to prefer a `src` subdirectory at the project root  ;)